_pages/relayimps/søstr.md

```diff
@@ -0,0 +1,12 @@
+---
+title: søstr
+permalink: /søstr/
+web: 
+github: metasikander/s0str
+tags:
+- relayImplementation
```

Comment:
Please rename the folder, too. It's just relays. The relay instances will not be handled here. They are already in https://github.com/Giszmo/nostr.info/blob/master/_data/relays.yml

```suggestion
- relay
```